pymc/distributions/multivariate.py

```diff
@@ -1631,11 +1592,95 @@ def logp(value, n, eta):
         )
 
 
-@_default_transform.register(LKJCorr)
+@_default_transform.register(_LKJCorr)
 def lkjcorr_default_transform(op, rv):
     return MultivariateIntervalTransform(floatX(-1.0), floatX(1.0))
 
 
+# Thin wrapper around _LKJCorr
```

Comment:
Comments like this have a tendency to get left behind. I would exclude or add it inside the class definition
```suggestion
```